src/main/java/org/jenkinsci/plugins/DependencyCheck/ResultAction.java

```diff
@@ -135,4 +143,47 @@ public JSONObject getSeverityDistributionJson() {
         return JSONObject.fromObject(getSeverityDistribution(), jsonConfig);
     }
 
+    public Api getApi() {
+        return new Api(this);
```

Comment:
in this case we have to return only severity distribution so you can return `new Api(getSeverityDistribution())` without add all other methods (getSeverityDistribution trigger a lock) and keep the buildaction clean